### ISSUE TITLE
Epochs, regularization, data augmentation

### ISSUE THREAD

**Issue by mansi1710** [mansi1710/Alzheimer-s-Disease-Prediction-using-3D-MRI-scans#3]
Vary epochs, learning rate, other parameters.
Try regularization, data augmentation to handle overfitting
